gradle/dependencies.gradle

```diff
@@ -43,6 +44,7 @@ ext {
     opentelemetryJaeger         : dependencies.create(group: 'io.opentelemetry', name: 'opentelemetry-exporters-jaeger', version: versions.opentelemetry),
     opentelemetryOtlp           : dependencies.create(group: 'io.opentelemetry', name: 'opentelemetry-exporters-otlp', version: versions.opentelemetry),
     opentelemetryZipkin         : dependencies.create(group: 'io.opentelemetry', name: 'opentelemetry-exporters-zipkin', version: versions.opentelemetryOther),
+    opentelemetryPrometheus     : dependencies.create(group: 'io.opentelemetry', name: 'opentelemetry-exporters-prometheus', version: versions.opentelemetry),
```

Comment:
@trask @anuraaga why there is `versions.opentelemetryOther` and `version.opentelemetry`?

Reply:
Jfrog has been well behaved recently but often some of the snapshot artifacts have different version suffixes. We've found there to be cases of three different suffixes for the same version. An annoyance when dealing with snapshots that will go away with GA so currently living with it.

Reply:
Which one should I use? 

Currently all point to the same versions
```
    opentelemetry       : '0.9.0-20200925.025016-49',
    opentelemetryOther  : '0.9.0-20200925.025016-49',
    opentelemetryAnother: '0.9.0-20200925.025016-49',
```

Reply:
Does not matter, `opentelemetry` is fine if it works now